FRAuth/FRAuth/Manager/KeychainManager.swift

```diff
@@ -226,6 +228,31 @@ struct KeychainManager {
         return nil
     }
     
+    //  MARK: - FROptions
+    
+    /// Returns current session's Token object that represents SSO Token
```

Comment:
Returns session's token?

Reply:
good catch, there is one more comment that needs fixing

Reply:
done